### PR TITLE
minor optimize

### PR DESCRIPTION
**Pull request by wolfboys**
minor optimize

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java [hunk 1]
```diff
@@ -69,27 +69,23 @@ public BackendClient(Routing routing, DorisReadOptions readOptions) {
...
             client = new TDorisExternalService.Client(protocol);
-            if (isConnected) {
-                logger.info("Success connect to {}.", routing);
-                return;
-            }
             try {
                 logger.trace("Connect status before open transport to {} is '{}'.", routing, isConnected);
                 if (!transport.isOpen()) {
                     transport.open();
                     isConnected = true;
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java:82]: maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

Comment (wolfboys) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java:82]: > maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

okay, thanks for your review

Comment (wolfboys) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java:82]: > maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

updated

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**